extensions/vscode/src/commands/install-cli.ts

```diff
@@ -0,0 +1,48 @@
+const cp = require("child_process");
+
+import { window, ProgressOptions } from "vscode";
+
+/**
+ * Installs Dart Frog CLI in the user's system if not already installed.
+ */
+export const installCLI = async (): Promise<void> => {
+  if (!hasDartFrogCliInstalled()) {
+    const options: ProgressOptions = {
+      location: 15,
+      title: "Installing Dart Frog CLI...",
+    };
+    window.withProgress(options, installDartFrogCliVersion);
+  }
+};
+
+/**
+ * Whether the user has Dart Frog CLI installed in their system.
+ *
+ * @returns {boolean} True if the user has Dart Frog CLI installed in their
+ * system, false otherwise.
+ */
+function hasDartFrogCliInstalled(): boolean {
+  try {
+    cp.execSync(`dart_frog --version`);
+    return true;
+  } catch (error) {
+    return false;
+  }
+}
+
+/**
+ * Installs Dart Frog CLI from pub.dev.
+ *
+ * @returns {Promise<void>} A promise that resolves when the installation is
+ * complete.
+ */
+async function installDartFrogCliVersion(): Promise<void> {
+  await cp.exec(
+    `dart pub global activate dart_frog_cli`,
```

Comment:
If we are running Dart, it is a good idea to add the dart extension as a dependency of this one on te manifest